api/tests/test_security_user_model.py

```diff
@@ -0,0 +1,46 @@
+from django.test import TestCase
+from ..models import SecurityUser
+
+
+class SecurityUserModelTest(TestCase):
+    """ Tests for the Security User Model """
+    def setUp(self):
+        SecurityUser.objects.create(
+            email='security@site.com',
+            cohort=0,
+            slack_handle='@test_security_user',
+            password='devpassword',
+            first_name="TestFirst",
+            last_name="TestLast",
+            phone_number="254720900900",
```

Comment:
i don't think security users will have cohort, email address and slack handle.. i'm thinking we can use `AbstractUser`, but modify `UserManager` to have `_create_securitydesk_user` for security users and  `_create_user` for other users

Reply:
Makes sense to modify `UserManager`. If we inherit from the User model, the email address is the username and it is required and I think it will almost always give an error.

Reply:
and if you specify the `USERNAME_FIELD ` for this model to be the phone number?

Reply:
I still the error `This field is required.`

Reply:
Have you pushed your latest changes?

Reply:
I have  now